test/unit/lib/plugins/aws/deployFunction.test.js

```diff
@@ -183,6 +184,138 @@ describe('AwsDeployFunction', () => {
     });
   });
 
+  describe('#KMSKeyArn should be set by "kmsKeyArn" or "awsKmsKeyArn"', () => {
```

Comment:
One last thing - let's move that describe into `describe` from https://github.com/serverless/serverless/pull/8697/files#diff-79623f798e186957caf38cdf7b770ba812c6a8bf7034dc4cb39abb234974e0aeR445 - we try to keep all `runServerless` based tests separately from the ones written the old way